cfgov/v1/models/molecules.py

```diff
@@ -12,121 +12,90 @@ def isRequired(field_name):
-            error_dict.update(e.params)
-            block_data = data
-
-        if not block_data['image'] and not block_data['image_path'] and not block_data['image_alt']:
-            img_err = ['Please upload or enter an image path']
-            error_dict.update({'image': img_err, 'image_path': img_err, 'image_alt': isRequired('Image alt')})
-
-        if block_data['image'] and block_data['image_path']:
-            img_err = ['Please select one method of image rendering']
-            error_dict.update({
-                'image': img_err,
-                'image_path': img_err})
-
-        if block_data['image_path'] and not block_data['image_alt']:
-            error_dict.update({'image_alt': isRequired('Image Alt')})
-
-        if error_dict:
-            raise ValidationError("ImageText2575 validation errors", params=error_dict)
-        else:
-            return block_data
-
     class Meta:
         icon = 'image'
-        template = 'v1/wagtail/molecules/image_text_2575.html'
+        template = '_includes/molecules/image-text-25-75.html'
 
 
 class TextIntroduction(blocks.StructBlock):
     heading = blocks.CharBlock(max_length=100, required=True)
     intro = blocks.CharBlock(max_length=100, required=True)
     body = blocks.RichTextBlock(required=False)
-    link_url = blocks.URLBlock(required=False)
-    link_text = blocks.CharBlock(max_length=100, required=False)
+    link = atoms.Hyperlink()
     has_rule = blocks.BooleanBlock(required=False)
 
     class Meta:
         icon = 'title'
-        template = 'v1/demo/molecules/text_introduction.html'
+        template = '_includes/molecules/text-introduction.html'
+
+
+class Hero(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
+    body = blocks.RichTextBlock(required=False)
+
+    image = atoms.ImageBasic()
+
+    background_color = blocks.CharBlock(max_length=100, required=False)
+    link = atoms.Hyperlink()
+    is_button = blocks.BooleanBlock(required=False)
+
+    class Meta:
+        icon = 'image'
+        template = '_includes/molecules/hero.html'
+
+
+class FormFieldWithButton(blocks.StructBlock):
+    btn_text = blocks.CharBlock(max_length=100, required=True)
+
+    required = blocks.BooleanBlock(required=False)
+    id = blocks.CharBlock(max_length=100, required=False)
+    info = blocks.RichTextBlock(required=False)
+    label = blocks.CharBlock(max_length=100, required=True)
+    type = blocks.CharBlock(max_length=100, required=True)
+    name = blocks.CharBlock(max_length=100, required=False)
+    placeholder = blocks.CharBlock(max_length=100, required=False)
+    attributes = blocks.CharBlock(max_length=100, required=False)
+
+    class Meta:
+        icon = 'mail'
+        template = '_includes/molecules/form-field-with-button.html'
 
 
 class CallToAction(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
     slug = blocks.CharBlock(required=True)
-    paragraph = blocks.RichTextBlock()
```

Comment:
@duelj @ajbush @jimmynotjim  [The specifications for the CTA](https://github.cfpb.gov/flapjack/Modules-V1/wiki/Standalone-CTA-\(with-button\)) say that this has paragraphs. Did this change or are the templates just not including those specs?